test/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,10 @@
 const { humanCatDogYears } = require("../src");
 
-// Look Ma, no handlebars!!!
+describe("humanCatDogYears", () => {
+  it("returns array of human, cat and dog years", () => {
+    expect(humanCatDogYears(1)).toEqual([1, 15, 15]);
```

Comment:
Would be good to get a couple more edge cases in here, such as passing in zero of a negative number. How should the function respond in cases like this?